### DISCUSSION TITLE
Live and Zoom Meetings in Draft?

### DISCUSSION THREAD

**Discussion by fbarnese** [code4recovery/12-step-meeting-list#1389] [General]
Hi folks.

At the www.oc-aa.org website, we just noticed that over 300 meetings were moved from Published to Draft.  I was looking for patterns ... and it appears that many of the ZOOM meeting URL's have been altered (deleting the meeting ID).  We still have the ID information noted in the URL notes.

It also appears that these ZOOM meetings now have been changed to attend in person, but no address appears.

Office staff says they haven't made any changes but to add and update.

Do we need to update each of these meetings manually?

Faith

**Comment by fbarnese** [comment]
Additional Comments:  I am trying to update a few records.  I notice that the Region is no longer in many of the records.  I've been able to publish a few.  However, most are not publishing.

**Reply by SIA-WEBSUP** [reply]
that seems like very bizarre behavior to happen on its own..

did you updated the TSML plugin recently?

incidentally, do you have a backups of your website? do you have a backups ( CSV downloads ) of your meetings?
are you noticing any other weirdness on your site?

Did you notice the if the ALL the meetings where you noticed the  'ZOOM meeting URL being altered' issue also had the 'status changed from Published to Draft' issue?

**Comment by fbarnese** [comment]
Have not updated the the TSML plugin.

Several of the meetings will not go back to published status when manually updated.

Faith Barnese
The Barnese Group, Inc
(480) 532-0007
***@***.***

Sent from my iPhone

On Mar 17, 2024, at 5:48 PM, SIA-WebSup ***@***.***> wrote:

﻿

that seems like very bizarre behavior to happen on its own..

did you updated the TSML plugin recently?

incidentally, do you have a backups of your website? do you have a backups ( CSV downloads ) of your meetings?
are you noticing any other weirdness on your site?

**Comment by SIA-WEBSUP** [comment]
The most common reason I've seen for the 'inability' to Publish a meeting is that the geocoding fails. 
When you enter a valid address for an in person meeting and tab away from the address field, does the address get updated? or is it cleared?

If the address is valid and the geocoding successful you should be able to Publish the meeting at that point.
If the address is invalid OR the geocoding fails for any other reason, the Publish button will stay gray.

if the meeting is an online meeting, it must a conference url ( the zoom link ).
if the meeting is an online meeting, for it to be displayed by the meeting guide app, you must supply a town or city and/or zip, but no street address.

can you check if maybe an invalid address is what is causing your problem?
